onnxscript/rewriter/generic_pattern.py

```diff
@@ -633,6 +634,10 @@ def make_pattern_rule(
         the rewriting rule
     """
 
+    warnings.warn(
+        "make_pattern_rule(...) is deprecated, use pattern.RewriteRule(...) instead",
+        DeprecationWarning,
```

Comment:
FutureWarning for deprecations: https://docs.python.org/3/library/exceptions.html#FutureWarning

Reply:
Thanks, changed